pkg/authorization/authorizer/scope/converter.go

```diff
@@ -106,18 +108,18 @@ func (userEvaluator) Validate(scope string) error {
 	return fmt.Errorf("unrecognized scope: %v", scope)
 }
 
-func (userEvaluator) Describe(scope string) string {
+func (userEvaluator) Describe(scope string) (description string, warning string, err error) {
```

Comment:
I don't use naked returns, but I like the named return documenting the values. Comments don't help tools like godoc and autocomplete